README.md

```diff
@@ -83,3 +83,24 @@ npm install -g git-cz
 ```
 
 To commit, just run `git-cz` in the terminal 😄
+
+
+### On SEB Windows computer
+
+Yarn support for our artifactory will be supported later on, but for now you need to logoff the VPN (right click the lock symbol in the tray).
+
+You start the 3 storybocks with thesse commands:
```

Comment:
Maybe add instructions for generic use as well eg.

```
npx nx run [project]:storybook
```

As we have more storybooks besides the 3 in the instructions.